sdk/openai/Azure.AI.OpenAI/src/Generated/AudioTranscriptionOptions.cs

```diff
@@ -34,15 +35,18 @@ public partial class AudioTranscriptionOptions
         /// If set to 0, the model will use log probability to automatically increase the temperature until certain thresholds are hit.
         /// </param>
         /// <param name="deploymentName"> The model to use for this transcription request. </param>
-        internal AudioTranscriptionOptions(BinaryData audioData, AudioTranscriptionFormat? responseFormat, string language, string prompt, float? temperature, string deploymentName)
+        internal AudioTranscriptionOptions(BinaryData audioData, string filename, AudioTranscriptionFormat? responseFormat, string language, string prompt, float? temperature, string deploymentName)
         {
             AudioData = audioData;
+            Filename = filename;
             ResponseFormat = responseFormat;
             Language = language;
             Prompt = prompt;
             Temperature = temperature;
             DeploymentName = deploymentName;
         }
+        /// <summary> The optional filename or descriptive identifier to associate with with the audio data. </summary>
+        public string Filename { get; set; }
```

Comment:
@trrwilson Taking into account XML documentation for this property, it doesn't look like it's optional. If I pass `null` or filename without extension, I will receive the following error message:
```json
"error": {
  "message": "Unrecognized file format. Supported formats: ['flac', 'm4a', 'mp3', 'mp4', 'mpeg', 'mpga', 'oga', 'ogg', 'wav', 'webm']",
  "type": "invalid_request_error",
  "param": null,
  "code": null
}
```

Maybe it would be better to specify it in XML doc that this property should have `{name}.{extension}` format. We can also list supported formats just for better visibility. Hope that makes sense!

Reply:
Thanks, this is great feedback and we'll make this clearer -- given there doesn't appear to be any use of the actual filename, it's tempting to simplify this to a format enumeration to sidestep the awkwardness of needing to contrive complete filenames. I need to look hard for a reason *not* to do that, but this'll get clarified one way or another.

Reply:
Sounds great, thank you!